Masonry/View.jsx

```diff
@@ -47,42 +48,42 @@
       static getHeightFromProps = (getState, props, columnSpan, columnGutter) => {
         return IMAGE_HEIGHT + TITLE_HEIGHT + FOOTER_HEIGHT;
       }
-      
+
       render() {
         ...
       }
     }
  */
 
-
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import throttle from 'lodash.throttle';
+import throttle from 'lodash/throttle';
```

Comment:
Is this because here we're relying on having all of lodash available?

Reply:
Probably just my lack of understanding the package setup, the `.` works when you've only installed the `throttle`-part? I installed the ent lib and didn't get the dot-syntax to work, so that's why the change.

Reply:
That's ok it's very ambiguous. You're right in that this was developed only with the `lodash.throttle` since that's the only function it needs. Let's keep it this way should this ever be packaged and distributed 👍